xml/esx-remove_existing_cluster_compute_resource_pool.xml

```diff
@@ -242,21 +242,24 @@ ansible-playbook -i hosts/verb_hosts neutron-stop --limit &lt;hostname&gt;;</scr
 | id                                   | alarm_definition_id                  | alarm_definition_name | metric_name       | metric_dimensions                         | severity | state | lifecycle_state | link | state_updated_timestamp  | updated_timestamp        | created_timestamp        |
 +--------------------------------------+--------------------------------------+-----------------------+-------------------+-------------------------------------------+----------+-------+-----------------+------+--------------------------+--------------------------+--------------------------+
 | cfc6bfa4-2485-4319-b1e5-0107886f4270 | cca96c53-a927-4b0a-9bf3-cb21d28216f3 | Host Status           | host_alive_status | service: system                           | HIGH     | OK    | None            | None | 2016-10-27T06:33:04.256Z | 2016-10-27T06:33:04.256Z | 2016-10-23T13:41:57.258Z |
-|                                      |                                      |                       |                   | cloud_name: entry-scale-kvm-esx-vsa-mml   |          |       |                 |      |                          |                          |                          |
+|                                      |                                      |
+|                   | cloud_name: entry-scale-kvm-esx-mml       |          |       |                 |      |                          |                          |                          |
```

Comment:
These tables seems [even] broken[er] now. Are these line breaks accidental?

Reply:
fixed. Yes they were accidental...or rather part of the intentional, helpful nature of emacs ensuring 78 character lines.